hazelcast/src/main/java/com/hazelcast/client/impl/clientside/ClusterDiscoveryServiceBuilder.java

```diff
@@ -139,6 +139,9 @@ private AddressProvider createAddressProvider(ClientConfig clientConfig, Discove
             String urlEndpoint = HazelcastCloudDiscovery.createUrlEndpoint(cloudUrlBase, discoveryToken);
             int connectionTimeoutMillis = getConnectionTimeoutMillis(networkConfig);
             HazelcastCloudDiscovery cloudDiscovery = new HazelcastCloudDiscovery(urlEndpoint, connectionTimeoutMillis);
+            //We use the usePublic parameter as true always because on the cloud context hazelcast nodes are never in the
+            //same network even-tough they can be in the same group/zone etc. For private addresses to be used, the
+            //members and clients should be all coordinated by the users so that private addresses can be used.
```

Comment:
what does this mean? 
```
//members and clients should be all coordinated by the users
```

Reply:
I tried to meant all the machines that contains clients and members needs to be opened by the same organization with same credentials to make private addresses work between them.

If this is is more clear, I will update the description accordingly.

Reply:
yes, please clarify it but even the explanation you put sounds very complex, can you simplify it? Why do you even need that sentence? Is the first sentence not enough?

Reply:
Ok, removing the second sentence.